src/assets/scripts/game/GameClockView.js

```diff
@@ -0,0 +1,96 @@
+import $ from 'jquery';
+import { digits_integer } from '../utilities/radioUtilities';
+import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
+
+/**
+ * @class GameClockView
+ */
+export default class GameClockView {
+    /**
+     * @for GameClockView
+     * @constructor
+     */
+    constructor() {
+        this.$element = null;
+        this.startTime = 0;
+        this.time = 0;
+
+        return this._init();
+    }
+
+    /**
+     * @for GameClockView
+     * @method _init
+     * @private
+     */
+    _init() {
+        this.$element = $(SELECTORS.DOM_SELECTORS.CLOCK);
+        this.$element.addClass(SELECTORS.CLASSNAMES.NOT_SELECTABLE);
+        this._setToCurrentZuluTime();
+
+        return this;
+    }
+
+    /**
+    * Updates the time stored in the clock
+    * @for GameClockView
+    * @method _tick
+    * @private
+    */
+    _tick() {
+        const elapsedTime = window.gameController.game.time * TIME.ONE_SECOND_IN_MILLISECONDS;
+        this.time = this.startTime + elapsedTime;
+    }
+
+    /**
+    * Updates the DOM with the new game time
+    * @for GameClockView
+    * @method _render
+    * @private
+    */
+    _render() {
+        const gameTime = window.gameController.game.time;
+        const clockDate = new Date(this.startTime + (gameTime * TIME.ONE_SECOND_IN_MILLISECONDS));
+        const hours = digits_integer(clockDate.getHours(), 2);
+        const minutes = digits_integer(clockDate.getMinutes(), 2);
+        const seconds = digits_integer(clockDate.getSeconds(), 2);
+        const clockTime = `${hours}:${minutes}:${seconds}`;
+        this.$element.text(clockTime);
```

Comment:
copied from slack:

the render method should only be concerned with that last line.  the rest of it should live in another method, `calculateClockTime()` or something that can also be called in the `update` method.  perhaps that method then sets a class property `this.clockTimeString` so the last line in render becomes: `this.$element.text(this.clockTimeString)`;